streams/stable.json

```diff
@@ -1,144 +1,157 @@
-                                "uncompressed-sha256": "ac254fe29c82cbcf92c16f6407717ad9cdf756543465e4d701e3a66ddcd347e4"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-azure.aarch64.vhd.xz",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-azure.aarch64.vhd.xz.sig",
+                                "sha256": "05fd0677c5a22553658361f3c0250ab85e7501a07397cadeedaaf0f82649d3eb",
+                                "uncompressed-sha256": "55598e18fc4dcec86fbabeb8dfdee699704245db4b35da9cdaafadbadcc19309"
                             }
                         }
                     }
                 },
                 "gcp": {
-                    "release": "41.20250302.3.2",
+                    "release": "41.20250315.3.0",
                     "formats": {
                         "tar.gz": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250302.3.2/aarch64/fedora-coreos-41.20250302.3.2-gcp.aarch64.tar.gz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250302.3.2/aarch64/fedora-coreos-41.20250302.3.2-gcp.aarch64.tar.gz.sig",
-                                "sha256": "850e320c41dd19fd92c48440336f6f53e0e412abcdf1906a1c641e20294fd63c",
-                                "uncompressed-sha256": "b0ef195e7c0e562b3469bacc11edb98eb492bd8fab588a41655c749fb98e35d8"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-gcp.aarch64.tar.gz",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-gcp.aarch64.tar.gz.sig",
+                                "sha256": "ef0e708c0812709c30292c1dd590b4668a5e06203a66f31d1ee3f5a841ee2b1e",
+                                "uncompressed-sha256": "65b521ac5f38a4939d59fad64f8ba2f5e4f30873364fc977f7ffe5e5a25e7f3a"
+                            }
+                        }
+                    }
+                },
+                "hetzner": {
+                    "release": "41.20250315.3.0",
+                    "formats": {
+                        "raw.xz": {
+                            "disk": {
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-hetzner.aarch64.raw.xz",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/41.20250315.3.0/aarch64/fedora-coreos-41.20250315.3.0-hetzner.aarch64.raw.xz.sig",
```

Comment:
The stream diff warnings here are expected because we are adding the `hetzner` platform to our outputs.